string new_ip
---
bool ip_changed